### ISSUE TITLE
[BUG - Release]Nothing happens after installing UE4SS

### ISSUE THREAD

**Issue by buendva** [UE4SS-RE/RE-UE4SS#439]
**Branch or Release**
UE4SS__v.3.0.1

**Game and Engine Version**
Palworld  

**Describe the bug**
When I put all the files extracted by UE4SS into the win64 folder and then started the game, nothing happened. It did not generate a LogicMods folder as mentioned in the tutorial. Even after I manually created a folder with the same name and put my mods in it, the game did not change at all, as if I did not put it in the win64 folder at all. Enter anything.

**Mods directory**
[Pal IVs-437-0-5-1708568047.zip](https://github.com/UE4SS-RE/RE-UE4SS/files/14694342/Pal.IVs-437-0-5-1708568047.zip)
This is the mod I tried to install. I downloaded it from nexus

**Screenshots, UE4SS Log, and .dmp file**
This is a screenshot of the win64 folder
![5c8addabda87fa675d4595a177761bd](https://github.com/UE4SS-RE/RE-UE4SS/assets/116340444/8844d40e-fbdd-4535-86e4-28510d95a578)

**Desktop (please complete the following information):**
 -win10

**Additional context**
I have tried changing bUseUObjectArrayCache in UE4SS-settings.ini to false, and also ensured that the value of BPModLoaderMod in mods.txt is 1. I have reinstalled the game and even reinstalled steam. Maybe the next step is to reinstall the system. If there is any other information you need me to provide, please leave me a message.

**Comment by ProfessorFuzzy** [comment]
ok so i'm having the same issue all the mods that require UE4SS So im almost sure its a UE4SS issue

**Comment by buendva** [comment]
> 好的，所以我在所有需要 UE4SS 的 mods 中都遇到了同样的问题，所以我几乎确定这是 UE4SS 问题

But I successfully installed the mod using the same method on another computer, so I'm not sure what went wrong.

**Comment by buendva** [comment]
> ok so i'm having the same issue all the mods that require UE4SS So im almost sure its a UE4SS issue

But I successfully installed the mod using the same method on another computer, so I'm not sure what went wrong.

**Comment by nexustheru** [comment]
perhaps check config and the eengine. opengl or dx

**Comment by buendva** [comment]
> perhaps check config and the eengine. opengl or dx

can you tell me how to check that？

**Comment by ProfessorFuzzy** [comment]
its weird because my case it was working just fine then i went to get on the other day and it just stopped for some reason mods like paldar mini map has stopped working

**Comment by UE4SS** [comment]
> its weird because my case it was working just fine then i went to get on the other day and it just stopped for some reason mods like paldar mini map has stopped working

Maybe the game updated and broke things.

**Comment by ProfessorFuzzy** [comment]
> > its weird because my case it was working just fine then i went to get on the other day and it just stopped for some reason mods like paldar mini map has stopped working
> 
> Maybe the game updated and broke things.

thats what im thinking

**Comment by nexustheru** [comment]
do you have any error log, from either game or this

**Comment by buendva** [comment]
> do you have any error log, from either game or uess4ss or the addon

Nothing at all

**Comment by ifalatik** [comment]
Same here, though on the XBox Version of Palword. If I can provide any files to help solve this I'd be happy to. I just installed UE4SS, with the default mods. From the logs I would expect F10 to open a console, though nothing happens.
[UE4SS.log](https://github.com/UE4SS-RE/RE-UE4SS/files/14856884/UE4SS.log)

**Comment by UE4SS** [comment]
> Same here, though on the XBox Version of Palword. If I can provide any files to help solve this I'd be happy to. I just installed UE4SS, with the default mods. From the logs I would expect F10 to open a console, though nothing happens. [UE4SS.log](https://github.com/UE4SS-RE/RE-UE4SS/files/14856884/UE4SS.log)

This doesn't appear to be the same problem.
You have a log file, and the original issue doesn't seem to generate a log file.
I'm not seeing anything wrong with your log file, UE4SS is injected and appears to be working.
As far as why F10 isn't bringing up the in-game console, I don't know, but since this is a separate problem, a new issue should be created.

**Comment by buendva** [comment]
> > Same here, though on the XBox Version of Palword. If I can provide any files to help solve this I'd be happy to. I just installed UE4SS, with the default mods. From the logs I would expect F10 to open a console, though nothing happens. [UE4SS.log](https://github.com/UE4SS-RE/RE-UE4SS/files/14856884/UE4SS.log)
> 
> This doesn't appear to be the same problem. You have a log file, and the original issue doesn't seem to generate a log file. I'm not seeing anything wrong with your log file, UE4SS is injected and appears to be working. As far as why F10 isn't bringing up the in-game console, I don't know, but since this is a separate problem, a new issue should be created.

Yes, I didn't find the log file. It should have been generated in the win64 folder, right? It seems that UE4SS was not injected successfully, but I don’t know which step is the problem?

**Comment by UE4SS** [comment]
> Yes, I didn't find the log file. It should have been generated in the win64 folder, right? It seems that UE4SS was not injected successfully, but I don’t know which step is the problem?

If UE4SS.log doesn't exist in the Win64 directory (if non-xbox), then UE4SS wasn't injected.
You can try manually injecting it.
Your directory structure looks correct.
The only possible problem I can think of is unicode characters in your path, those might be a problem, I can't recall.

**Comment by BATTINGCLOCKS** [comment]
i’m having the same problem, i also don’t have a log file. i’ve tried to inject UE4SS a couple times to no avail.

**Comment by technic-Angel6644** [comment]
I may have found the solution to the issue by accident, it appears that adding the game folder as a drive in protontricks is necessary, for write permissions/access to be allowed. When I looked into the logfiles created by oblivion, it directly references the S: drive, which is what I mounted the steamapps/common folder as.

`[2025-05-07 20:51:31.0490371] Console created
[2025-05-07 20:51:31.0491018] UE4SS - v3.0.1 Beta #0 - Git SHA #437a8ff
[2025-05-07 20:51:31.0491089] UE4SS Build Configuration: Game__Shipping__Win64 (MSVC)
[2025-05-07 20:51:31.0523227] Setting up mods...
[2025-05-07 20:51:31.0527663] Starting mods (from mods.txt load order)...
[2025-05-07 20:51:31.0528855] Starting mods (from enabled.txt, no defined load order)...
[2025-05-07 20:51:31.0531066] No specific game configuration found, using default configuration file
[2025-05-07 20:51:31.0531161] Config: S:\common\Oblivion Remastered\OblivionRemastered\Binaries\Win64\ue4ss\UE4SS-settings.ini

[2025-05-07 20:51:31.0531210] root directory: S:\common\Oblivion Remastered\OblivionRemastered\Binaries\Win64\ue4ss
[2025-05-07 20:51:31.0531251] working directory: S:\common\Oblivion Remastered\OblivionRemastered\Binaries\Win64\ue4ss
[2025-05-07 20:51:31.0531291] game executable directory: S:\common\Oblivion Remastered\OblivionRemastered\Binaries\Win64
[2025-05-07 20:51:31.0531430] game executable: S:\common\Oblivion Remastered\OblivionRemastered\Binaries\Win64\OblivionRemastered-Win64-Shipping.exe (160878392 bytes)


[2025-05-07 20:51:31.0531477] mods directory: S:\common\Oblivion Remastered\OblivionRemastered\Binaries\Win64\ue4ss\Mods
[2025-05-07 20:51:31.0531517] log directory: S:\common\Oblivion Remastered\OblivionRemastered\Binaries\Win64\ue4ss
[2025-05-07 20:51:31.0531557] object dumper directory: S:\common\Oblivion Remastered\OblivionRemastered\Binaries\Win64\ue4ss


[2025-05-07 20:51:31.0542225] DebugGame Setting Enabled? false
[2025-05-07 20:51:31.0577329] PS Scan attempt 1
[2025-05-07 20:51:31.0577504] [PS] Reading image
[2025-05-07 20:51:31.0578120] [PS] Starting scan
[2025-05-07 20:51:31.3283931] [PS] Scan finished in 270.5574ms
[2025-05-07 20:51:31.3284209] [PS] Found GUObjectArray: 0x149145160
[2025-05-07 20:51:31.3284271] [PS] Found GMalloc: 0x149064c60
[2025-05-07 20:51:31.3284321] [PS] Found FName::ToString: 0x140f61010
[2025-05-07 20:51:31.3284365] [PS] Found FName::FName(wchar_t*): 0x140f45a70
[2025-05-07 20:51:31.3284408] [PS] Found StaticConstructObject_Internal: 0x141129d30
[2025-05-07 20:51:31.3284458] [PS] Found FText::FText(FString&&): 0x140e561f0
[2025-05-07 20:51:31.3284501] [PS] Found FUObjectHashTables::Get(): 0x14111eef0
[2025-05-07 20:51:31.3284545] [PS] Found GNatives: 0x149144580
[2025-05-07 20:51:31.3284588] [PS] Found ConsoleManagerSingleton: 0x140e0ff00
[2025-05-07 20:51:31.3284638] PS scan successful`
